core/src/main/scala-2/caliban/schema/SchemaDerivation.scala

```diff
@@ -64,6 +64,7 @@ trait SchemaDerivation[R] extends LowPriorityDerivedSchema {
           Some(getName(ctx)),
           getDescription(ctx),
           ctx.parameters
+            .filter(p => !p.annotations.exists(_ == GQLExcluded()))
```

Comment:
How about `.filterNot(_.annotations.exists(_ == GQLExcluded()))`?

Reply:
👍 didn't know that existed!

Reply:
Don't like `p => p` => `_`? 😆

Reply:
Haha sure! I sometimes try to shy away from multiple levels of `_` but I think it's fine in this case!